tests/console/check_system_info.pm

```diff
@@ -81,6 +81,20 @@ sub check_buildid {
     }
 }
 
+# SLE-21783: Update nodejs-common for SLE15 SP4
+# check the nodejs16 is releaseed to SLE15 SP3 & SLE15 SP4
+# check the nodejs-common points to nodejs16
+# test steps:
+# 1) install nodejs-common on SLES15SP3 and migrated SLES15SP4 when wsm is available in SCC_ADDONS
+# 2) check nodejs16 is installed together with nodejs-common
+sub check_nodejs_common {
+    record_info('SLE-21783', 'check nodejs-common');
+    zypper_call('in nodejs-common', log => 'zypper.log');
+    die "nodejs-common doesn't point to nodejs16 as default" unless (script_run('grep -E "nodejs16" /tmp/zypper.log') == 0);
+    script_run('rpm -qa | grep nodejs16');
```

Comment:
See my first comments, just grep nodejs16 can't verify for the feature.
Maybe you need verify the installed package include nodejs16 and nodejs-common?

#####################################
i+ | nodejs-common        | package | 5.0-150400.1.1      | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool
   | nodejs-default       | package | 5.0-150400.1.1      | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool
   | nodejs-devel-default | package | 5.0-150400.1.1      | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool
i  | nodejs16             | package | 16.13.2-150400.11.2 | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool

Reply:
> See my first comments, just grep nodejs16 can't verify for the feature. Maybe you need verify the installed package include nodejs16 and nodejs-common?
> 
> ##################################### i+ | nodejs-common | package | 5.0-150400.1.1 | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool | nodejs-default | package | 5.0-150400.1.1 | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool | nodejs-devel-default | package | 5.0-150400.1.1 | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool i | nodejs16 | package | 16.13.2-150400.11.2 | x86_64 | SLE-Module-Web-Scripting15-SP4-Pool

yes, agree with you, add the installation checking.